### ISSUE TITLE
I suggest to extract these infos from world.json

### ISSUE THREAD

**Issue by daily2432121** [Razhil/GoWCardFilter#1]
Instead of hard-coded them in js

**Comment by Razhil** [comment]
You're right. I was thinking about extracting those from mongoDB. Or are you suggesting that world.json is already existing somewhere ? ^^'

**Comment by daily2432121** [comment]
You can find it here, if you have installed steam and GoW game through steam

C:\Program Files (x86)\Steam\steamapps\common\Gems of War\res\Data\

**Comment by daily2432121** [comment]
BTW.
I made this real time drop rate web site if you are interested.
http://gow.mengmeizi.com/Calculator

**Comment by Razhil** [comment]
Damn, that's very convenient, I should have checked before putting everything in mongo. Wasted quite some time there. Oops :(

I'll update based on the world.js, that makes it very easy for me to update the site after each update of GoW too. 

Thanks a lot :) I'll have a look at your calculator ;)
